hypothesis-python/tests/cover/test_reproduce_failure.py

```diff
@@ -103,9 +103,9 @@ def test_errors_with_did_not_reproduce_if_the_shape_changes():
     n = len(b)
 
     @reproduce_failure(__version__, encode_failure([b]))
-    @given(st.binary(min_size=n + 1, max_size=n + 1))
-    def test(x):
-        assert x == b
+    @given(st.binary(min_size=n, max_size=n) | st.integers())
```

Comment:
What's going on with this test change?

Reply:
This is a bit of a latent fix uncovered by this pull. Providing `b"a"` to `draw_bytes(2, 2)` is a misalignment and gets replaced with the zero-index choice `b"\x00\x00`, which actually still does reproduce the failure. To invoke DidNotReproduce we need to run the failure to either an overrun or something valid (just something different than status.interesting). This test used to overrun directly, but after this change it does not. I changed the @given to something that would overrun the given failure.